src/client/pages/ToeristischeVerhuurDetail/VakantieVerhuurList.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  ToeristischeVerhuur,
+  ToeristischeVerhuurVergunning,
+  ToeristischeVerhuurVergunningaanvraag,
+} from '../../../server/services';
+import {
+  addTitleLinkComponent,
+  SectionCollapsible,
+  Table,
+} from '../../components';
+import { useAppStateGetter } from '../../hooks';
+import styles from './ToeristischeVerhuurDetail.module.scss';
+
+const DISPLAY_PROPS_VERHUUR = {
+  dateStart: 'Start verhuur',
+  dateEnd: 'Einde verhuur',
+  dateRequest: 'Ontvangen op',
+  duration: 'Aantal nachten',
+};
+
+const filterVerhuur = (vergunning: ToeristischeVerhuurVergunningaanvraag) => {
+  return (v: ToeristischeVerhuurVergunning) => {
+    if (
+      !['Vergunning bed & breakfast', 'Vergunning vakantieverhuur'].includes(
+        v.title
+      ) &&
+      v.location === vergunning.location &&
+      v.dateStart &&
+      vergunning.dateStart &&
+      new Date(v.dateStart) >= new Date(vergunning.dateStart) &&
+      v.dateEnd &&
+      vergunning.dateEnd &&
+      new Date(v.dateEnd) <= new Date(vergunning.dateEnd)
+    ) {
+      return v;
+    }
+    return undefined;
+  };
+};
+
+const VakantieVerhuurList = ({
+  vergunning,
+}: {
+  vergunning: ToeristischeVerhuurVergunningaanvraag;
+}) => {
+  const { TOERISTISCHE_VERHUUR } = useAppStateGetter();
+  const { content } = TOERISTISCHE_VERHUUR;
+
+  const verhuur = addTitleLinkComponent(
+    (content?.vergunningen?.map(filterVerhuur(vergunning)) || []).filter(
+      Boolean
+    ),
+    'dateStart'
+  ) as ToeristischeVerhuur[];
+
+  const cancelledVerhuur = verhuur.filter(
+    (v) => v?.title === 'Geannuleerde verhuur'
+  );
+
+  const plannedVerhuur = verhuur.filter((v) => v?.title === 'Geplande verhuur');
+
+  const previousVerhuur = verhuur.filter(
+    (v) => v?.title === 'Afgelopen verhuur'
+  );
+
+  const isVergunningActiveAndValid = (
+    vergunning: ToeristischeVerhuurVergunningaanvraag
+  ) => {
+    return (
+      vergunning.isActual &&
+      !['Ingetrokken', 'Geweigerd'].includes(vergunning.status)
+    );
+  };
+
+  if (vergunning.status === 'Geweigerd') {
+    return <></>;
+  }

```

Comment:
Als wij niks tonen met deze status, waarom kunnen we er dan heen navigeren?

Reply:
Mogelijk moeten we dit er in de backend al uitfilteren?

Reply:
Dit component toont de lijst(en) met geplande/geannuleerde/afgelopen verhuren en staat op de detail pagina van een vergunning. Ik ben er vanuit gegaan dat bij een vergunning met status `Geweigerd` deze items (verhuren) niet aanwezig zijn. Het lijkt me echter wel zinnig om de detail pagina van de vergunning zelf te kunnen zien i.v.m de documenten die je mogelijk kunt/wilt downloaden etc.